### DISCUSSION TITLE
Animations!

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [JakeStanger/ironbar#123] [Show and tell]
[bar.webm](https://user-images.githubusercontent.com/68589851/233814607-e8e74d10-0314-4c2c-8675-b4e19866f157.webm)

This is awesome!
I can put so much in my bar without cluttering it\

(https://gitea.com/SimplyKyle/dotfiles/src/branch/main/home/.config/ironbar)

**Comment by JakeStanger** [comment]
Very cool, thanks for sharing!
